src/domain/creator/register/validateData.ts

```diff
@@ -0,0 +1,31 @@
+
+import validator, { ValidationSchema } from '^/integrations/validation/module';
+
+import { optionalIdValidation } from '^/domain/definitions';
+
+import InvalidCreator from '../InvalidCreator';
+import { fullNameValidation } from '../definitions';
+import { ValidationModel } from './types';
+
+const schema: ValidationSchema = {
+    fullName: fullNameValidation,
+    email:
+    {
+        message: 'Invalid email',
```

Comment:
```suggestion
        message: 'Value is not an valid email',
```

This version is more consist with the others.